Gopkg.toml

```diff
@@ -36,8 +36,8 @@ required = [
 
 [[constraint]]
   name = "github.com/knative/pkg"
-  # HEAD as of 2018-08-02
-  revision = "51828370417a6cce0cf7b271a5ee49bf3b01734c"
+  source = "https://github.com/scothis/pkg.git"
```

Comment:
/hold 

We should fix this before we merge.

Reply:
yep, that was just to get CI to run the code, and why I put a `/hold` when opening the PR :)